### ISSUE TITLE
About the Model Accuracy

### ISSUE THREAD

**Issue by FantasticCHH** [dusty-nv/jetson-inference#1365]
I have a simple question for create the Model Accuracy Image to record my training.
I haven't understand how to create this Image and save it
![part1](https://user-images.githubusercontent.com/63339132/154511929-5947dbc9-a11c-453b-b5a4-2c0db12c5393.png)
![part2](https://user-images.githubusercontent.com/63339132/154511943-529d5097-b03c-433f-bfa9-5d9b5618088b.png)
![part3](https://user-images.githubusercontent.com/63339132/154511950-78e6b554-bd1a-4d6c-9ace-c714ae60b3fd.png)

**Comment by dusty-nv** [comment]
Hi @FantasticCHH, at the end of each epoch you can get the training/validation accuracy from the AverageMeter [`top1.avg` ](https://github.com/dusty-nv/pytorch-classification/blob/393914b484fc946876e0a1fe37c971acdfb1fec6/train.py#L448) value.   Then store these in list(s) and make the plot at the end.
